### ISSUE TITLE
[BUG] Image_text_matching crashes if tuning data is None

### ISSUE THREAD

**Issue by rxjx** [autogluon/autogluon#3864]
Ubuntu 20, autogluon 1.0.0

If no tuning data is specified in the call to fit() for image-text matching, then an error is reported from generate_train_test_split_combined() since there's no label column. Specifying a label column seems to flip the problem type to 'multiclass' internally when fit() is being executed even if the problem_type was specified as 'image_text_matching' in the MultiModalPredictor ctor.
```

from autogluon.multimodal import MultiModalPredictor

predictor = MultiModalPredictor(query='lbl', response='img', problem_type='image_text_similarity', verbosity=4)

predictor.fit(train_data=train_df)
```

**Comment by ArijitSinghEDA** [comment]
@rxjx Thank you for this. I am using Autogluon 1.1.0 and facing the same issue. I was baffled by the fact why it required a label column, since it worked fine when no label was being passed in Autogluon 0.8.2
